docs/installation.rst

```diff
@@ -46,3 +48,13 @@ GCM-Filters and submit issues_ and `pull requests`_.
 .. _xarray: http://xarray.pydata.org
 .. _issues: https://github.com/ocean-eddy-cpt/gcm-filters/issues
 .. _`pull requests`: https://github.com/ocean-eddy-cpt/gcm-filters/pulls
+
+
+How to run the example notebooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you want to run the example notebooks in this documentation, you will need a few extra dependencies that you can install via::
+
+   mamba env create -f docs/environment.yml
```

Comment:
I just noticed that we suggest conda elsewhere. It might be better to do all examples with conda and make a note somewhere that mamba can be used as a faster drop in replacement?